packages/ui/src/routes/bridge/BridgeConfirmPage.tsx

```diff
@@ -229,6 +237,27 @@ const BridgeConfirmPage: FunctionComponent<{}> = () => {
         ? hasNativeAssetBalance
         : hasNativeAssetBalance && hasFromTokenBalance
 
+    const nativeTokensInDestinationNetworkStatus =
+        useAddressHasEnoughNativeTokensToSend(
+            bridgeQuote.bridgeParams.params.toChainId,
```

Comment:
`bridgeQuote.bridgeParams.params` is being used many times in this file, wouldn't make it easier to read if we access the needed properties in specific vars? i.e.:
```typescript
const { toChainId: destinationChainId, toToken: destinationToken ... } = bridgeQuote.bridgeParams.params